src/launcher/nix-launch.ts

```diff
@@ -94,9 +95,12 @@ export default async function launch(config: ConfigWithRuntime, osConfig: OsConf
             args.push('--v=1');
             args.push('--attach-console');
         }
+        if (typeof config.services !== undefined && config.services != null) {
```

Comment:
This should only run in mac/nix right right? Is there a guard?

Reply:
the only guard is in the fact that only the nix-launcher uses config.services, I believe we added this here to prevent double manifest downloads?  @pbaize can you refresh my memory here